mediaire_toolbox/data_cleaner.py

```diff
@@ -75,9 +132,22 @@ def clean_up(self, dry_run=False):
                     "Current total size is too big (%s bytes), need to delete some data to free up space" % current_size)
             if delete:
                 removed.append(folder)
+                pre_clean_size = self.current_size(folder)
                 if dry_run:
                     default_logger.info('(dry-run) Would remove folder [%s]' % folder)
+                    if self.whitelist or self.blacklist:
+                        self.clean_folder(folder)
```

Comment:
Shouldn't the dry_run parameter also be passed to clean_folder? Otherwise the dry_run wouldn't work